### PR TITLE
Added get_raw_transaction() and get_raw_transaction_testnet()

### PR DESCRIPTION
**Pull request by Gz75y45ms3kc**
This just includes the two new functions:
get_raw_transaction()
get_raw_transaction_testnet()
I executed "run_tests.py" and there were no errors (but I didn't add any tests for the new functions).

### CODE REVIEW

File: bitcash/network/services.py [hunk 1]
```diff
@@ -204,6 +206,22 @@ def get_unspent_testnet(cls, address):
             for tx in r.json()['utxos']
         ]
 
+    @classmethod
+    def get_raw_transaction(cls, txid):
+        r = requests.get(cls.MAIN_RAW_API.format(txid), timeout=DEFAULT_TIMEOUT)
+        if r.status_code != 200:  # pragma: no cover
```
Comment (merc1er) [inline bitcash/network/services.py]: I would like to replace lines 212 -> 213 with `r.raise_for_status()  # pragma: no cover` 😀

File: bitcash/network/services.py [hunk 2]
```diff
@@ -204,6 +206,22 @@ def get_unspent_testnet(cls, address):
...
+            raise ConnectionError
```
Comment (merc1er) [inline bitcash/network/services.py]: I would like to replace lines 212 -> 213 with r.raise_for_status() # pragma: no cover 😀

File: bitcash/network/services.py [hunk 3]
```diff
@@ -204,6 +206,22 @@ def get_unspent_testnet(cls, address):
...
+        response = r.json(parse_float=Decimal)
+        return response
+
+    @classmethod
+    def get_raw_transaction_testnet(cls, txid):
+        r = requests.get(cls.TEST_RAW_API.format(txid), timeout=DEFAULT_TIMEOUT)
+        if r.status_code != 200:  # pragma: no cover
+            raise ConnectionError
```
Comment (merc1er) [inline bitcash/network/services.py]: Same thing here

**Comment by teran-mckinney** [comment]
> It returns `ConnectionError: All APIs are unreachable.` if we pass a wrong transaction ID. Maybe it should display the error we get on the API directly (400, `parameter 1 must be hexadecimal string`)?

I tend to agree, but that logic may be trickier than it sounds. Since it's going through the connection pooling and retrying, that might be why it throws the ConnectionError. If we can validate the TX ID before hand, that would make for the easiest validation, although wouldn't tell us if the TX was never on the network (and in that case, it might be in mempool for some APIs, but not others).

File: bitcash/network/services.py (hunk 2 above)
Comment (Gz75y45ms3kc) [reply bitcash/network/services.py]: That's fine with me.

File: bitcash/network/services.py (hunk 3 above)
Comment (Gz75y45ms3kc) [reply bitcash/network/services.py]: Likewise, fine with me.

**Comment by Gz75y45ms3kc** [comment]
More informative error messages are something I can get behind, but can't offer any insight into implementing them.

**Comment by teran-mckinney** [comment]
I think we can figure out something for more informative error messages and the txid validation. This is good with me. Can you replace the if status code, raise ConnectionError with r.raise_for_status()?

Thanks for getting back to us.

**Pull request merged**